src/plone/restapi/tests/test_dxcontent_serializer.py

```diff
@@ -498,6 +499,26 @@ def test_allow_discussion_obj_instance_allows_global_enabled(self):
         self.assertIn("allow_discussion", obj)
         self.assertEqual(True, obj["allow_discussion"])
 
+    def isPlone61OrAbove(self):
+        """Check if the Plone version is 6.1 or above."""
+        installer = get_installer(self.portal)
+        return installer.getVersion() >= "6.1"
+
+    def test_allow_discussion_portal_default(self):
+        """Not globally addable, not fti enabled, not obj instance enabled"""
+        if self.isPlone61OrAbove():
+            # Test for Plone 6.1 and above
+            self.portal.invokeFactory("Document", id="doc2")
+            serializer = getMultiAdapter(
+                (self.portal.doc2, self.request), ISerializeToJson
+            )
+            obj = serializer()
+
+            self.assertIn("allow_discussion", obj)
+            self.assertEqual(False, obj["allow_discussion"])
+        else:
+            self.skipTest("Test skipped for Plone versions earlier than 6.1")
```

Comment:
You can skip the test like this:

https://github.com/plone/plone.restapi/blob/d5b83a67b5a894b7985c67e6c2ae0c9eb1034fd6/src/plone/restapi/tests/test_serializer_catalog.py#L52